### PR TITLE
16x speed of this repo with 16x rocket ships

### PR DESCRIPTION
**Pull request by OrenLeung**
🚀🚀🚀🚀

### CODE REVIEW

**Comment by OrenLeung** [comment]
@LemonFace0309 @eden-chan @jimmyygit plz approval my critical PR

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
 
-# Getting Started
+# Getting Started 🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀
```
Comment (jimmyygit) [inline README.md:3]: There are 16x rocket ships here. That is 6x too many according to the title.

**Review by jimmyygit — CHANGES_REQUESTED** [review]
Not impressed.

**Comment by OrenLeung** [comment]
resolved @jimmyygit suggestions and edited the PR title

File: README.md (hunk 1 above)
Comment (mabelene) [inline README.md:3]: LOL what is happening

**Review by LemonFace0309 — CHANGES_REQUESTED** [review]
you can do better than just 16x...

**Review by eden-chan — APPROVED** [review]
LGTM!

**Comment by OrenLeung** [comment]
@jimmyygit is @LemonFace0309's jeff dean 

[who could be yr 1000x engineer?](https://www.theengineeringmanager.com/growth/who-could-be-your-jeff-dean/)

File: README.md (hunk 1 above)
Comment (OrenLeung) [reply README.md:3]: @mabelene plz approve and merge "LGTM"

**Review by mabelene — APPROVED** [review]
vercel doesn't work but other than that LGTM loll

**Review by jimmyygit — APPROVED** [review]

**Review by LemonFace0309 — APPROVED** [review]
We got 22 rocket ships instead of 16. @OrenLeung you're overachieving is respectable.

**Pull request merged**